### PR TITLE
Feats: Boon Focus & Bane Focus

### PR DESCRIPTION
**Pull request by ucffool**
#193

### CODE REVIEW

File: feats/feats.yml [hunk 1]
```diff
@@ -383,7 +383,7 @@
     You are specialized in the use a particular boon that is iconic to your character.
   effect: |
     <ul>
-    <li><strong>Tier 1</strong> - When you invoke the chosen boon on a single target, you succeed automatically and do not need to make an action roll. If the invocation has multiple targets, success is not automatic, but you get advantage 2 on the action roll to invoke the boon.</li>
+    <li><strong>Tier 1</strong> - You can invoke the boon at any of the power levels you could access via your other means. When you invoke the chosen boon on a single target, you succeed automatically and do not need to make an action roll. If the invocation has multiple targets, success is not automatic, but you get advantage 2 on the action roll to invoke the boon.</li>
```
Comment (brianfeister) [inline feats/feats.yml]: Thanks @ucffool, sorry to nitpick but I think the caveat belongs in the middle, right after

> ... automatically and do not need to make an action roll.

Comment (ucffool) [inline feats/feats.yml]: I know that's what you meant, but when I reread it I realized it broke up the sentence flow. It flows from describing a single-target to describing multi-target nicely, and the phrase being placed prior to both has the same effect without breaking up the flow.
If you still disagree I'm happy to change it, but that was my rational.

Comment (ucffool) [inline feats/feats.yml]: I read the boon being discussed in the _description_ section, thus it makes sense in my reading.

Comment (brianfeister) [inline feats/feats.yml]: Yeah so `description` and `effect` should be treated separately, description normally should not even reference mechanics. From a certain perspective, that's the actual problem here. But either way, one should not depend on the other.

Comment (ucffool) [inline feats/feats.yml]: Ah. See, little style guide things in your head and nowhere else. ;)

Comment (brianfeister) [inline feats/feats.yml]: Indeed. We'll get there.

**Comment by brianfeister** [comment]
@ucffool Sorry clobbered your change with another fix.

**Comment by ucffool** [comment]
@brianfeister done.

**Comment by brianfeister** [comment]
💯

**Pull request merged**